src/osp/Active/SysVehicleSync.cpp

```diff
@@ -0,0 +1,239 @@
+/**
```

Comment:
No change needed for this PR. But wanted to ask your opinion on using SPDX copyright headers instead of full license text in very file?

Reply:
SPDX? sounds like some kind of pushbutton switch. I'd have to read up.

Replacing the header entirely doesn't appear to be the right way to go:

> When a license defines a recommended notice to attach to files under that license (sometimes called a “standard header”), the SPDX project recommends that the standard header be included in the files, in addition to an SPDX ID.
> 
> Additionally, when a file already contains a standard header or other license notice, the SPDX project recommends that those existing notices should not be removed. The SPDX ID is recommended to be used to supplement, not replace, existing notices in files.

https://spdx.dev/ids/#how